src/runtime/platform/NativeCodePage.cs

```diff
@@ -15,7 +15,7 @@ class NativeCodePageHelper
         /// Gets the operating system as reported by python's platform.system().
         /// </summary>
         [Obsolete]
-        public static string OperatingSystemName { get; private set; }
+        public static string OperatingSystemName => PythonEngine.Platform;
```

Comment:
Since this is marked `[Obsolete]`, and we are bumping major version, let's remote it altogether.